<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Half-Life Model Viewer [Qt]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="pnlUpper" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>400</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabBottom">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabModel">
        <attribute name="title">
         <string>Model</string>
        </attribute>
        <widget class="QCheckBox" name="chkRenderTextured">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Render textured</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkShowBones">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show bones</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkShowAttachments">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>10</y>
           <width>211</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show attachments</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>191</width>
           <height>81</height>
          </rect>
         </property>
         <property name="title">
          <string>Wireframe options:</string>
         </property>
         <widget class="QCheckBox" name="chkWireframeModel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Wireframe model</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="chkWireframeoverlay">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>171</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Wireframe overlay</string>
          </property>
         </widget>
        </widget>
        <widget class="QCheckBox" name="chkShowHitBoxes">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>50</y>
           <width>211</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show hitboxes</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkShowNormals">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>70</y>
           <width>211</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show normals</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>10</y>
           <width>341</width>
           <height>151</height>
          </rect>
         </property>
         <property name="title">
          <string>Model info:</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Textures count:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Body parts count:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblModelName">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lblTexturesCount">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>30</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblBodypartsCount">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>50</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblSkinsCount">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Skins count:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblBonesCount">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>90</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Bones count:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblAttachmentsCount">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>110</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Attachments count:</string>
          </property>
         </widget>
         <widget class="QLabel" name="lblSequencesCount">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>130</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sequences count:</string>
          </property>
         </widget>
        </widget>
        <widget class="QCheckBox" name="chkShowBonesNames">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show bones names</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkShowAttachmentsNames">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>30</y>
           <width>211</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Show attachments names</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoneControllers">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>100</y>
           <width>181</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QSlider" name="sliderBoneControllerValue">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>150</y>
           <width>160</width>
           <height>16</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Controller value</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="lblBoneControllerBoneName">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>130</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Bone</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabBody">
        <attribute name="title">
         <string>Body</string>
        </attribute>
        <widget class="QListWidget" name="lstBodyParts">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>230</width>
           <height>141</height>
          </rect>
         </property>
        </widget>
        <widget class="QListWidget" name="lstBodySubModels">
         <property name="geometry">
          <rect>
           <x>259</x>
           <y>30</y>
           <width>250</width>
           <height>141</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>10</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Body parts:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>261</x>
           <y>10</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Body part model:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>234</x>
           <y>90</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;-&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QListWidget" name="lstSkins">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>30</y>
           <width>250</width>
           <height>141</height>
          </rect>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>524</x>
           <y>30</y>
           <width>3</width>
           <height>141</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>542</x>
           <y>10</y>
           <width>231</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Skins:</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabTextures">
        <attribute name="title">
         <string>Textures</string>
        </attribute>
        <widget class="QListWidget" name="lstTextures">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>481</width>
           <height>170</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="btnExportTexture">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>20</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Export as...</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkIsChromeTexture">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>50</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Is chrome</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblTextureSize">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>0</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0 x 0 pixels</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkIsAdditiveTexture">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>70</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Is additive</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkIsMaskedTexture">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>90</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Is masked</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="spinImageZoom">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>145</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Zoom:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabSequences">
        <attribute name="title">
         <string>Sequences</string>
        </attribute>
        <widget class="QListWidget" name="lstSequences">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>256</width>
           <height>171</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="lblSequenceName">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>20</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>idle1</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblSequenceFPS">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>40</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>15 FPS</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblSequenceFrames">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>60</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>31 frames</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblSequenceEvents">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>80</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0 events</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>0</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Events list:</string>
         </property>
        </widget>
        <widget class="QListWidget" name="lstEvents">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>20</y>
           <width>161</width>
           <height>151</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="lblEventFrame">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>20</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Frame 0</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblEventEvent">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>40</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Event 0</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblEventType">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>60</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Type 0</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblEventOptions">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>80</y>
           <width>171</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_models">
     <property name="title">
      <string>Recent models</string>
     </property>
    </widget>
    <addaction name="actionLoad_model"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_models"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad_model">
   <property name="text">
    <string>Load model...</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
